README.md

```diff
@@ -98,11 +98,11 @@ You are very welcome to communicate with the developers and users of Dolphin Sch
 
 ### Contributor over time
 
-[![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/dolphinscheduler)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/dolphinscheduler)
+[![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/dolphinscheduler)]
```

Comment:
Maybe we should totally delete it, WDYT?
```suggestion
```

Reply:
Done.